Cargo.toml

```diff
@@ -1,12 +1,14 @@
 [workspace]
 resolver = "2"
-members = ["celestia", "node", "proto", "rpc", "types"]
+members = ["celestia", "node", "proto", "rpc", "types", "wasm-node"]
 
 [workspace.dependencies]
 celestia-node = { version = "0.1.0", path = "node" }
 celestia-proto = { version = "0.1.0", path = "proto" }
 celestia-rpc = { version = "0.1.0", path = "rpc", default-features = false }
 celestia-types = { version = "0.1.0", path = "types", default-features = false }
+celestia-wasm-node = { version = "0.1.0", path = "wasm-node" }
```

Comment:
nitpick, I think I'd prefer `celestia-node-wasm` and `node-wasm` but I have no strong opinion, wdyt? @oblique @fl0rek

Reply:
I think wasm-node is less popular overall,  so renamed to `node-wasm` and `celestia-node-wasm`.